en/contributing/build.md

```diff
@@ -220,17 +220,22 @@ The main steps are:
    cd DroneCore
    git submodule update --init --recursive
    ```
-1. Enter the following command in your host's terminal:
-   ```sh
-   docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
-   ```
+1. Enter either of the following command in your host's terminal:
```

Comment:
These just replace the old target `dronecore/dronecore` with `dronecore/dronecore-fedora-27`. The problem is that they don't work - new bash prompt is empty/not mapped to the host tree. 

I tried the following:
```
docker run --rm -it -v $HOME/DroneCore:/root/DroneCore:rw dronecore/dronecore-ubuntu-16.04 bash
```
That worked (ie the directory was mapped properly) but ownership of files is by `root`, so no access in the parent. 

What is the right way to run these two?

Reply:
Yes, the ownership problem is a general issue with docker and a bit tricky. I usually just do a `sudo make clean` outside of docker afterwards, or I remember to do `make clean` before leaving docker.

Reply:
PS NOte that this may also be why the files are mapped to -v `$HOME/DroneCore:/home/docker1000/src/DroneCore:rw` - because root/Dronecore belongs to root.

Reply:
Oh, that's right, it is now `make distclean` to remove the actual folders.

Reply:
> Could we update the scripts to support passing in the current user? Px4 docker allows you to pass in the current user ID as --env=LOCAL_USER_ID="$(id -u)" - this is then used to create the files. You can see this being loaded into the environment here

I would like to do it a bit cleaner than that but I need to look into it.
I made an issue for now: #254